package.json

```diff
@@ -37,11 +37,11 @@
     "pify": "^2.2.0",
     "rimraf": "^2.4.3",
     "run-applescript": "^2.0.0",
-    "xdg-trashdir": "^2.0.0"
+    "xdg-trashdir": "^2.1.0",
+    "path-exists": "^2.0.0"
```

Comment:
@tborychowski It's also because if we didn't, it would cause unrelated diff noise for the next commit that adds a dependency, because this dependency would be sorted by npm.